src/components/calcite-tab-title/calcite-tab-title.scss

```diff
@@ -1,124 +1,113 @@
-$tab-margin: 1.25rem;
-
 :host {
-  flex: 0 1 auto;
-  outline: none;
-  margin-right: $tab-margin;
-  margin-inline-start: 0;
-  margin-inline-end: $tab-margin;
+  @apply flex-initial outline-none mx-5;
+  margin-inline-start: theme("margin.0");
+  margin-inline-end: theme("margin.5");
 }
 
 :host([layout="center"]) {
+  @apply text-center my-0 mx-5;
   flex-basis: 200px;
```

Comment:
Perhaps a design follow up - but it would be cool if we could update this to an even int rem.

Reply:
how about 12rem? `flex-basis: theme("spacing.48");`